examples/cart/index.js

```diff
@@ -118,39 +118,78 @@ $(function() {
     $('.buy-button').on('click', function (event) {
       event.preventDefault();
       var id = product.selectedVariant.id;
-      addVariantToCart(product.selectedVariant, 1);
+      variantToCartHandler(product.selectedVariant, 1);
+      setPreviousFocusItem(this);
+      $('#checkout').focus();
     });
   }
 
   /* Increase product variant quantity in cart
   ============================================================ */
   function attachQuantityIncrementListeners(product) {
-    $('.cart').on('click', '.quantity-increment', function() {
+    var $DomNode = $(this);
```

Comment:
Confused about the value of `this` here... wouldn't it just be the global object?